migrations/env.py

```diff
@@ -10,8 +10,6 @@
 
```

Comment:
Why are you modifying this file?
This exclude stuff was hard to get set up correctly to avoid issues with the scheduler.